wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md

```diff
@@ -3,9 +3,9 @@ title: Resumable Upload API
 ---
 
 # Resumable Upload API
-This article demonstrates how to use the response object from the [Generate File Resumable Upload Url](wix-media-v2/files/generatefileresumableuploadurl) endpoint to upload a file to a site's Media Manager. 
+This article demonstrates how to use the response object from `generateFileResumableUploadUrl()` to upload a file to a site's Media Manager. 
```

Comment:
Restore the links. The only place they need to be removed from is from the reference itself (ie., the service.json files). MD files can still have links.